COMPONENT_GUIDE.md

```diff
@@ -1,317 +1,4 @@
-- [Globally available components](#globally-available-components)
-  - [Video](#video)
-    - [Usage](#usage)
-  - [Intro](#intro)
-    - [Usage](#usage-1)
-  - [Steps](#steps)
-    - [Usage](#usage-2)
-  - [Step](#step)
-    - [Usage](#usage-3)
-- [A code example](#a-code-example)
-- [Another code example](#another-code-example)
-  - [Code blocks](#code-blocks)
-    - [Usage](#usage-4)
-  - [Callouts](#callouts)
-    - [Usage](#usage-5)
-  - [Related Resources](#related-resources)
-    - [Maximum references](#maximum-references)
-    - [Resource population logic](#resource-population-logic)
-    - [Order of priority](#order-of-priority)
-    - [Resource site tags](#resource-site-tags)
-  - [Announcement Banner](#announcement-banner)
+# Component Documentation
 
-# Globally available components
-
-## Video
-
-`<Video />` component provides formatting for videos in markdown.
-
-### Usage
-
-The Video component requires two props:
-
-- `id`: the video ID
-- `type`: the host of the video. Accepted values are:
-  - `wistia`
-  - `youtube`
-
-```md
-<Video id="zxunt1u1as" type="wistia"/>
-```
-
-## Intro
-
-The `<Intro />` component provides formatting for the title and introduction of the markdown document.
-
-### Usage
-
-It takes the title provided in the Frontmatter and accepts plain text for the description. An example of Frontmatter that will have a title of **Example Guide**:
-
-```
----
-path: '/example'
-duration: '30 min'
-title: 'Example Guide'
-template: 'GuideTemplate'
-description: 'Example guide page'
----
-```
-
-It also accepts a `<Video />` component as a child, which it will place on the left side of the description.
-
-```md
-<Intro>
```

Comment:
The `Intro` is only available on the dev site. We should probably keep that around.

Reply:
oh... ok. why not put them in the global theme?